cves/kernel/CVE-2022-1671.yml

```diff
@@ -149,10 +148,16 @@ discovered:
 
     If there is no evidence as to how this vulnerability was found, then please
     explain where you looked.
-  answer:
-  automated:
-  contest:
-  developer:
+  answer: |
+    A problem was found in the way the Linux operating system
+    handles certain actions related to network communication. Specifically, when a
+    security protocol isn't specified, the part of the system responsible for
+    managing network connections can misbehave. This could allow a person with
+    access to the system to either crash it or potentially access information
+    stored in the system's memory that they shouldn't be able to see.
```

Comment:
Maybe add a line at the beginning explaining that someone found this? I just wonder how they found it or maybe they were just randomly exploring the kernel's code lol